src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -358,15 +358,27 @@ public boolean isNativeMethod() {
      */
     @Override
     public String toString() {
-        int estimatedLength = length(classLoaderName) + 1
+        return appendTo(
+                new StringBuilder(
+                        estimatedLength()))
+                .toString();
+    }
+
+    int estimatedLength() {
```

Comment:
Don't think you need to split this into a new method; you can just keep the old variable as the calculation is too long.